soepy/python/solve/solve_auxiliary.py

```diff
@@ -211,19 +216,15 @@ def pyth_backward_induction(attr_dict, state_space_args):
 
 
 def construct_covariates(
-    states_all, states_number_period, max_states_period, attr_dict
+    states_all, states_number_period, max_states_period, model_params
```

Comment:
It appears you had the convention of passing in `model_params` as the first argument. Why not here?